src/BodyFunctionality/BodyPartGenerator.java

```diff
@@ -0,0 +1,382 @@
+package BodyFunctionality;
+
+import javafx.scene.paint.Color;
+
+import java.util.ArrayList;
+
+public class BodyPartGenerator
+{
+
```

Comment:
These Enums should probably be broken into their own files in a folder

Reply:
I was thinking about that but didn't figure out how to do it yet.

Reply:
Make the folder, the. If you right click and create new class you can select Enum from the menu.